sessionctx/variable/session.go

```diff
@@ -75,6 +75,9 @@ var (
 	enableAdaptiveReplicaRead uint32 = 1
 )
 
+// ConnStatusShutdown indicates that the connection status is closed by server.
+const ConnStatusShutdown int32 = 2
```

Comment:
Cloud you add some comments why this value must to be `2` to avoid modify it by mistake in the future?